test/integration/test-pipe.js

```diff
@@ -36,7 +37,7 @@ var server = http.createServer(function(req, res) {
       var field = FIELDS.shift();
       assert.strictEqual(name, field.name);
       assert.strictEqual(file.name, path.basename(field.value.path));
-      assert.strictEqual(file.type, mime.lookup(file.name));
+      assert.strictEqual(file.type, mime.lookup(file.name) || defaultContentType);
```

Comment:
One way would be to have hash of filename and expected content-types, since we're testing for them. Something like:

```
{
  'my_buffer': FormData.DEFAULT_CONTENT_TYPE,
  'my_file': mime.lookup(common.dir.fixture + '/unicycle.jpg'),
  'remote_file': mime.lookup(common.dir.fixture + '/unicycle.jpg')
}
```